Calibration/LumiAlCaRecoProducers/plugins/AlcaPCCEventProducer.cc

```diff
@@ -37,7 +37,8 @@ class AlcaPCCEventProducer : public edm::global::EDProducer<> {
 
 private:
   const edm::InputTag pixelClusterLabel_;
-  const std::string trigstring_;  //specifies the trigger Rand or ZeroBias
+  const std::string trigstring_;      //specifies the trigger Rand or ZeroBias
+  const bool savePerROCInfo_ = true;  // save per ROC data (important for the special fills)
```

Comment:
what's the point of setting this `const` variable as true if you need to changed later on?